src/index.ts

```diff
@@ -56,6 +56,7 @@ for (let i = 0; i < filesLoadedBeforeTrace.length; i++) {
 interface TopLevelConfig {
   enabled: boolean;
   logLevel: number;
+  clsMechanism: 'none'|'auto';
```

Comment:
Can you use `ContextPropagationMechanism` type here instead of repeating the type?